test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -0,0 +1,114 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2017 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with
+# Altair.
```

Comment:
I'm not sure if you need to have the paragraph above.

Reply:
I actually borrowed it from other existing PTL script, so I am reluctant to change anything here unless there is a ticket which mentions this change is needed in all the files.

Reply:
Seems fine to me.  This matches the license text posted at:
https://pbspro.atlassian.net/wiki/spaces/DG/pages/16973843/License+Text+for+Headers

Reply:
I have updated the text per changes in PP-1174.